### ISSUE TITLE
Support set custom extraData for Attachment from FileUploader

### ISSUE THREAD

**Issue by donlan** [GetStream/stream-chat-android#4690]
e.g.

```
override fun sendImage(
        channelType: String,
        channelId: String,
        userId: String,
        file: File,
        callback: ProgressCallback,
    ): Result<UploadedImage> {
        // uploading ...
       // val extra = some data from upload api

       // UploadedImage.extraData would be set to Attachment.extraData
        return UploadedImage(file = it.file, extraData = mapOf(extra))
    }
```

**Comment by filbabic** [comment]
@donlan Fair enough!

I'll look into this with the team to see if the extraData can be exposed here and accepted on the BE side.

**Comment by donlan** [comment]
> @donlan Fair enough!
> 
> I'll look into this with the team to see if the extraData can be exposed here and accepted on the BE side.

Thanks!

**Comment by JcMinarro** [comment]
Hello @donlan 
We already added `extraData` to `UploadedFile` class.
It was implemented on #5230 and it is already available on our last release.
